langcfgs/zh_HK.langcfg

```diff
@@ -22,3 +22,19 @@
 ; with HarTex. If not, see <https://www.gnu.org/licenses/>.
 
 LanguageIdentifier: zh_HK
+
+; Section: Global Plugin, About Command
+GlobalPlugin.AboutCommand.EmbedDescription: 由 HarTex 團隊維護及開發, HarTex 係一個針對 Discord 高效管理去進行咗優化嘅 Bot.
+GlobalPlugin.AboutCommand.EmbedBotVersionFieldName: Bot 嘅版本
+GlobalPlugin.AboutCommand.EmbedWhitelistedGuildsFieldName: Whitelist 咗嘅 Server
+
+; Section: Global Plugin, Ping Command
+GlobalPlugin.PingCommand.InitialResponse: Hello! 有咩幫到你? :eyes:
```

Comment:
additionally translate `Hello!` phrase as well.

Reply:
You can decide on this.